src/onegov/feriennet/forms/attendee.py

```diff
@@ -318,7 +318,7 @@ def populate_tos(self) -> None:
 
         layout = DefaultLayout(self.model, self.request)
 
-        self.accept_tos.description = Markup(render_macro(
+        self.accept_tos.description = Markup('{}').format(render_macro(
```

Comment:
Yeah, that doesn't work with `chameleon.utils.Markup`, that particular class only has a `__html__` method, nothing else, so you can't use `format` with it.

Actually you should just be able to use `render_macro` directly, since it return `markupsafe.Markup` now, no need to wrap it.